### DISCUSSION TITLE
能提供离线文档吗？

### DISCUSSION THREAD

**Discussion by srarlight** [umijs/umi#9006] [Q&A]
公司内网 访问不了外网文档，我看ant-design有个ant-design-gh-pages分支 支持sgo本地启动

**Comment by ahKevinXy** [comment]
package.json 里面不是有个吗

**Comment by srarlight** [comment]
这个文档并不能完全离线啊 依赖包打包压缩放到内网环境也是无法启动 依然需要安装依赖



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: ahKevinXy ***@***.***&gt;
发送时间: 2022年8月18日 17:37
收件人: umijs/umi ***@***.***&gt;
抄送: starlight ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [umijs/umi] 能提供离线文档吗？ (Discussion #9006)





 
package.json 里面不是有个吗
